docs/src/guides/drupal9/redis.md

```diff
@@ -1,149 +1,172 @@
 ---
 title: "Using Redis with Drupal 9.x"
 sidebarTitle: "Redis"
-description: |
-    Add Redis caching to your existing Drupal site.
+description: Add Redis caching to your existing Drupal site.
 weight: -70
 ---
 
-If you are using the Platform.sh-provided Drupal template, most of this work is already done for you.
-Redis is already configured and is enabled after the installation is complete.
+Redis is a fast open-source in-memory database and cache, 
+useful for application-level caching. 
+For more information on this service, see [our dedicated Redis page](../../add-services/redis.md) 
+or the [official Redis documentation](https://redis.io/docs/).
 
-Note that this `redis` service is ephemeral, meaning it doesn't persist if the container moves or is shut down.
-Your app must treat it as ephemeral and not rely on it being there.
+The following steps let you:
```

Comment:
:hammer_and_wrench: https://developers.google.com/style/lists#introductory-sentences-for-lists
```suggestion
Follow the instructions on this page to do one of the following:
```

I'm not in love with that wording, but the idea is something that works as a standalone sentence.